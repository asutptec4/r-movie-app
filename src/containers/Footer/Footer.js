import React from 'react';

import styles from './Footer.module.scss';

const Footer = () => {
  return (
    <footer className={styles.footer}>
      <span className="app-logo">NetflixRoulette</span>
    </footer>
  );
};

export default Footer;
