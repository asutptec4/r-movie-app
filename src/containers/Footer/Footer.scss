@import '../../scss/index.scss';

.footer {
  background-color: $footer-color;
  padding: 15px;
  text-align: center;
}
