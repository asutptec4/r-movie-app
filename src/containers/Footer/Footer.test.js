import React from 'react';
import renderer from 'react-test-renderer';

import Footer from './Footer';

describe('<Footer />', () => {
  const component = renderer.create(<Footer />);

  test('render', () => {
    expect(component).toMatchSnapshot();
  });
});
