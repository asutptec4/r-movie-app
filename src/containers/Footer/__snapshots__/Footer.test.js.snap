// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> render 1`] = `
<footer
  className="footer"
>
  <span
    className="app-logo"
  >
    NetflixRoulette
  </span>
</footer>
`;
